src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
+        return Files.readAllLines(Paths.get(filePath));
```

Comment:
Consider using try-with-resources to ensure that the file resources are closed properly after the operation is completed. This is a good practice to prevent potential resource leaks.